src/utils.py

```diff
@@ -79,3 +80,56 @@ def chain_id(self) -> int:
         Aligned with https://chainlist.org/
         """
         return {Network.MAINNET: 1, Network.GNOSIS: 100}[self]
+
+
+class LoweredString(fields.String):
+    """Custom marshmallow String field for lowered string"""
+
+    def _deserialize(self, value, *args, **kwargs):
+        if hasattr(value, "lower"):
+            value = value.lower()
+        return super()._deserialize(value, *args, **kwargs)
+
+    def _serialize(self, value, attr, obj, **kwargs):
+        if value is None:
+            return ""
+        return str(value).lower()
```

Comment:
In both of these methods I see several unused attributes... (attr, obj, args, kwargs) are these required by the Schema interface? How might one go about handling the lint/type errors that arise from this?

Reply:
They were part of the suggested way to implement custom Fields in marshmallow: https://marshmallow.readthedocs.io/en/stable/custom_fields.html#creating-a-field-class

That's a good question. Not sure how this is possible. I'll try removing them and see if it still works.